+nosnoc/+discrete_time_problem/Stewart.m

```diff
@@ -908,8 +919,140 @@ function create_solver(obj, solver_options, plugin)
                     end
                 end
             end
+            
+            if ~isempty(active_set)
+                [IG,IH,~] = obj.process_active_set(active_set);
+            else
+                IG = [];
```

Comment:
please add  one line comment what happens to the solver if empty empty I_G and I_H are passed.